app/src/main/java/com/alexandruvalentinconstantin/newsreader/feature/newslist/listener/OnArticleSelected.java

```diff
@@ -0,0 +1,14 @@
+package com.alexandruvalentinconstantin.newsreader.feature.newslist.listener;
+
+import android.annotation.SuppressLint;
+
+import com.alexandruvalentinconstantin.newsreader.feature.newslist.model.ArticleItemViewModel;
+
+public interface OnArticleSelected {
+
+    @SuppressLint("CheckResult")
+    void onDeleteItemSelected(ArticleItemViewModel item);
```

Comment:
we probably don't probably need to the onDeleteItemSelected, maybe we can give rid of it